packages/node-opcua-samples/bin/simple_client_ts.ts

```diff
@@ -455,7 +455,7 @@ async function main() {
     console.log(" --- NAMESPACE ARRAY ---");
     const namespaceArray = dataValue.value.value;
     for (const namespace of namespaceArray) {
-        console.log(" Namespace ", namespace.index, "  : ", namespace);
+        console.log(" Namespace ", namespaceArray.indexOf(namespace), "  : ", namespace);
```

Comment:
Please revert, the intend is really  `namespace.index`

Reply:
I am getting `undefined` when i call `namespace.index` but i would expect the Indexnumber.
Why could that be?

Reply:
sorry,I confirmed ,  you are right.
 namespaceArray returns a bunch of string not a array of Namespace Object.